providers/base/units/optical/test-plan.pxu

```diff
@@ -16,6 +16,7 @@ _name: Optical drive tests (Manual)
 _description:
  Optical drive tests (Manual)
 include:
+ optical/detect                                 certification-status=blocker
  optical/read_.*                                certification-status=blocker
  optical/bluray-read_.*                         certification-status=blocker
```

Comment:
Do these tests also need to be run after suspend? They currently are not.

Reply:
I used my laptop to do list-bootstrapped but it doesn't have CDROM.
Here is another one with CD ROM
https://pastebin.canonical.com/p/2nvG6ZcCMY/